composer.json

```diff
@@ -25,7 +25,7 @@
         "symfony/monolog-bundle"               : "^3.0",
         "symfony/polyfill-apcu"                : "^1.0",
         "symfony/swiftmailer-bundle"           : "^2.3",
-        "symfony/symfony"                      : "^3.2",
+        "symfony/symfony"                      : "dev-master",
```

Comment:
I agree. I think `lint:xliff` command will be available in Symfony 3.3 so let's just wait for it. Otherwise, I can remove `xliff` command to make this PR mergeable for now - we may add xliff linter later when it will be released. Wdyt?